src/base/QXmppMessage.h

```diff
@@ -26,6 +26,8 @@
 #ifndef QXMPPMESSAGE_H
 #define QXMPPMESSAGE_H
 
+#include "QXmppMixInvitation.h"
```

Comment:
this can be forward declared

Reply:
I tried it but it does not build when I use `class QXmppMixInvitation;`.

Reply:
I tested it and it works for me, what's the error message?

Reply:
I had to add `#include "QXmppMixInvitation.h"` to `tests/qxmppmessage/tst_qxmppmessage.cpp`. bbf612a6b294bc73dcf9a741b9ce78b4c63f6b3a should fix it.